app/lucy/src/app/components/base-form/waypoint/waypoint-info/waypoint-info.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="content">
+  <!-- Section 1 -->
+  <h3 class="section-header">
+    What is a Way-point?
+  </h3>
+  <div class="section-content">
+    This tool allows the user to define an observation by generating a path (of a defined width/offset value)
+    and captures an area along a linear corridor. This is usually used to mark  observations of invasive species
+    that occur along roads and rivers. It can be used to properly define the area spatially.
+  </div>
+  <div class="section-content">
+    The interface allows the user to enter points based on their UTM location i.e. Easting and Northing values.
+  </div>
+  <div class="section-content">
+    Upon completion, the user clicks to proceed and generate a map. A buffer is created and displayed on a map.
+  </div>
+
+  <!-- Section 2 -->
+  <h3 class="section-header">
+    Please note:
+  </h3>
+  <div class="section-content">
+    - Points must be sequential to generate a map that is exactly as recorded on ground.
+  </div>
+  <div class="section-content">
+    <p>
+      - Minimum of 2 points are required to create a linear way-point path.
+    </p>
+    <p>
+      - Maximum of 10 points can be recorded.
```

Comment:
Isn't the the maximum number of points for the waypoint is set to 20?

Reply:
Damn it. Yes. Mike requested that we up the max to 20 yesterday. Ahhh the perils of hard-coding.

Good catch!

Reply:
Yah, I think we can reduce the risk of hardcoding errors like these by moving up the constant values to a common file from where we can use it anywhere so that there is a single source of truth and we can update it easily if needed

Reply:
Partially fixed - I updated the hard-coded values, but they're still hard-coded. There's a few of them in this file, so removing the hard-coding is a bigger fix than what I have time for right now.